proto/cosmos/bank/v1beta1/query.proto

```diff
@@ -354,6 +361,25 @@ message QueryDenomOwnersResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
+// QueryDenomOwnersByQueryRequest defines the request type for the DenomOwnersByQuery RPC query,
+// which queries for a paginated set of all account holders of a particular
+// denomination.
+message QueryDenomOwnersByQueryRequest {
```

Comment:
Missing since comment.

Reply:
I think we don't need `since comment`  because we have to back port this query to previous versions  which have `DenomOwners` query

Reply:
Are we backporting this at the first place? It will be backported to v0.50 only (as it's a feature), so we should have a since 0.50.3 (or 4, depending when we merge this).